Readme.md

```diff
@@ -71,14 +123,30 @@ write_chunk(
 
 The written file can now be read in Python via
 
+### Polars
```

Comment:
How about just adding a "### Usage from python section" and then include the polars / pandas distinction in paragraph form. This way there would be less headings. At the moment, the ratio of heading to content seems a bit unbalanced to me.